source/etherpad.rst

```diff
@@ -0,0 +1,194 @@
+
+.. code-block:: console
+ :emphasize-lines: 3
+
+  //IP and port which etherpad should bind at
+  "ip": "0.0.0.0",
+  "port" : 9001,
+
+You also need to set up the MySQL_ databse settings, therefore you should completely replace these codeblocks:
+
+::
+
+  //The Type of the database. You can choose between dirty, postgres, sqlite and mysql
+  //You shouldn't use "dirty" for for anything else than testing or development
+  "dbType" : "dirty",
+  //the database specific settings
+  "dbSettings" : {
+                   "filename" : "var/dirty.db"
+                 },
+
+  /* An Example of MySQL Configuration
+   "dbType" : "mysql",
+   "dbSettings" : {
+                    "user"    : "root",
+                    "host"    : "localhost",
+                    "password": "",
+                    "database": "store",
+                    "charset" : "utf8mb4"
+                  },
+  */
+
+with the following. Watch out to replace ``<username>`` with your username (2 times) and ``<mysql_password>`` with your password that you looked up in the prerequisites:
+
+.. code-block:: console
+ :emphasize-lines: 4,6,7
+ 
+  //MySQL Configuration
+  "dbType" : "mysql",
+  "dbSettings" : {
+                    "user"    : "<username>",
+                    "host"    : "localhost",
+                    "password": "<mysql_password>",
+                    "database": "<username>_etherpad",
+                    "charset" : "utf8mb4"
+                  },
+
+Setup .htaccess
+---------------
+
+.. include:: includes/proxy-rewrite.rst
+
+Setup daemon
+------------
+
+Create ``~/etc/services.d/etherpad.ini`` with the following content:
+
+.. warning:: Replace ``<username>`` with your username!
+
+.. code-block:: ini
+
+ [program:etherpad]
+ command=/home/<username>/etherpad/bin/run.sh
+
+In our example this would be:
+
+.. code-block:: ini
+
+ [program:etherpad]
+ command=/home/etherpad/etherpad/bin/run.sh
+
+Tell supervisord_ to refresh its configuration and start the service:
+
+::
+
+ [etherpad@stardust ~]$ supervisorctl reread
+ etherpad: available
+ [etherpad@stardust ~]$ supervisorctl update
+ etherpad: added process group
+ [etherpad@stardust ~]$ supervisorctl status
+ etherpad                            RUNNING   pid 26020, uptime 0:03:14
+ [etherpad@stardust ~]$ 
+
+If it's not in state RUNNING, check your configuration.
+
+Best practices
+==============
+
+Personalization
+---------------
+
+Take a deeper look into the ``~/etherpad/settings.json``, you still might want to adjust the title or the welcoming text of new created pads. If you want to use plugins, you will also need to set up a admin account there.
+
+Updates
+=======
+
+.. note:: You need to keep Etherpad Lite updated by pulling the current sourcecode from Github with ``git pull origin``. Afterwards, use ``supervisorctl restart etherpad`` to restart with the new codebase.
+
+
+.. _Loremipsum: https://en.wikipedia.org/wiki/Lorem_ipsum
```

Comment:
That's a placeholder ☺️

Reply:
I found a feed though: https://github.com/ether/etherpad-lite/releases.atom

So maybe add a note with the correct feed above?

```
.. note:: Check the update feed_ regularly to stay informed about the newest version.
```